### ISSUE TITLE
Dataset class

### ISSUE THREAD

**Issue by unofgithub** [lschmidtke/shape_templates#7]
Hello,

It would be really helpful if you could upload you dataset.py (not the dataset itself). Could you please let me know if you could upload the dataset.py you used? Thank you!

**Comment by lschmidtke** [comment]
Hi, the dataset class will not really help, since it simply loads preprocessed frames. However, I will be uploading the h36m preprocessing code instead soon (soon meaning over christmas because I need to clean it up).

**Comment by unofgithub** [comment]
Hello,

Could you please let me know if the preprocessing code and dataset code will be uploaded soon? Thank you so much!

**Comment by lschmidtke** [comment]
Hi! 
sorry for the late response. I finally found the time to clean and upload the processing code. Have a look in the `processing` folder.

Cheers,
Luca

**Comment by unofgithub** [comment]
Thank you so much!! In the dataset class, could you please let me know how I should be selecting the pair of frames to return in the Dataset class to reproduce the pretrained model that you uploaded?

Also, could you please let me know if the configuration values in default.yaml are the ones you used to train the pretrained model you uploaded?

**Comment by lschmidtke** [comment]
Hi,

once you have preprocessed the dataset with the provided script, all frames in a particular folder for each subject will contain the same background, so you can simply randomly sample two or more frames from a given folder.

I'm not sure regarding the exact parameters, but anchor and boundary loss weight between 0.5 and 1 should work.
I haven't really finetuned parameters like learning rate or batch size, but the provided checkpoint was trained with an lr of 1e-4.

Cheers,
Luca
